pipelines/skylab/snM3C/snM3C.wdl

```diff
@@ -62,105 +62,69 @@ workflow snM3C {
-            allc_uniq_reads_stats = unique_reads_allc.allc_uniq_reads_stats,
-            unique_reads_cgn_extraction_tbi = unique_reads_cgn_extraction.output_tbi_tar,
+            allc_uniq_reads_stats = unique_reads_allc_and_cgn_extraction.allc_uniq_reads_stats,
+            unique_reads_cgn_extraction_tbi = unique_reads_allc_and_cgn_extraction.extract_allc_output_tbi_tar,
             plate_id = plate_id
     }
 
     output {
         File MappingSummary = summary.mapping_summary
-        Array[File] trimmed_stats = Sort_and_trim_r1_and_r2.trim_stats_tar
-        Array[File] r1_trimmed_fq = Sort_and_trim_r1_and_r2.r1_trimmed_fq_tar
-        Array[File] r2_trimmed_fq = Sort_and_trim_r1_and_r2.r2_trimmed_fq_tar
-        Array[File] hisat3n_stats_tar = Hisat_3n_pair_end_mapping_dna_mode.hisat3n_paired_end_stats_tar
-        Array[File] hisat3n_bam_tar = Hisat_3n_pair_end_mapping_dna_mode.hisat3n_paired_end_bam_tar
-        Array[File] unique_bam_tar = Separate_unmapped_reads.unique_bam_tar
-        Array[File] multi_bam_tar = Separate_unmapped_reads.multi_bam_tar
-        Array[File] unmapped_fastq_tar = Separate_unmapped_reads.unmapped_fastq_tar
-        Array[File] split_fq_tar = Split_unmapped_reads.split_fq_tar
-        Array[File] merge_sorted_bam_tar = Hisat_single_end_r1_r2_mapping_dna_mode_and_merge_sort_split_reads_by_name.merge_sorted_bam_tar
-        Array[File] name_sorted_bams = merge_original_and_split_bam_and_sort_all_reads_by_name_and_position.name_sorted_bam
-        Array[File] pos_sorted_bams = merge_original_and_split_bam_and_sort_all_reads_by_name_and_position.position_sorted_bam
-        Array[File] remove_overlap_read_parts_bam_tar = remove_overlap_read_parts.output_bam_tar
-        Array[File] dedup_unique_bam_and_index_unique_bam_tar = dedup_unique_bam_and_index_unique_bam.output_tar
-        Array[File] unique_reads_cgn_extraction_allc = unique_reads_cgn_extraction.output_allc_tar
-        Array[File] unique_reads_cgn_extraction_tbi = unique_reads_cgn_extraction.output_tbi_tar
-        Array[File] chromatin_contact_stats = call_chromatin_contacts.chromatin_contact_stats
+        Array[File] name_sorted_bams = merge_original_and_split_bam_and_sort_all_reads_by_name_and_position_and_deduplicate.name_sorted_bam
+        Array[File] unique_reads_cgn_extraction_allc= unique_reads_allc_and_cgn_extraction.allc
+        Array[File] unique_reads_cgn_extraction_tbi = unique_reads_allc_and_cgn_extraction.tbi
+        Array[File] unique_reads_cgn_extraction_allc_extract = unique_reads_allc_and_cgn_extraction.extract_allc_output_allc_tar
+        Array[File] unique_reads_cgn_extraction_tbi_extract = unique_reads_allc_and_cgn_extraction.extract_allc_output_tbi_tar
         Array[File] reference_version = Hisat_3n_pair_end_mapping_dna_mode.reference_version
+        Array[File] chromatin_contact_stats = call_chromatin_contacts.chromatin_contact_stats
```

Comment:
Are there files being tarred here that shouldn't be included? Should we just output the two file types that they are looking for? This might be a question for Brian too.

Reply:
So i opened up this chromatin_contact_stats tar file, and it is filled with files ending in `hisat3n_dna.all_reads.contact_stats.csv `. This is not on Wubin's output list. 

It looks like he wants `.hisat3n_dna.all_reads.3C.contact.tsv.gz` and `.hisat3n_dna.all_reads.dedup_contacts.tsv.gz`. I will add these 2 outputs as final outputs. 

It does look like we still need the `hisat3n_dna.all_reads.contact_stats.csv` files as it is input to the summary task, so i will keep that one but will not set it as a final output. Let me know if this makes sense to you!

Reply:
This plan sounds good. Thank you.